.circleci/config.yml

```diff
@@ -156,10 +156,7 @@ ubuntu_settings: &ubuntu_settings
     - run:
         name: Install prerequisites
         command: |
-          apt-get update
-          apt-get install -y software-properties-common
-          apt-get remove -y libnss-systemd
```

Comment:
I'd prefer those removed as a separate commit, or at least specifically mentioned in the commit message as no longer needed. Took us quite some time to figure it out in the first place.